src/index.html

```diff
@@ -1,13 +1,403 @@
+              src="./images/Logo.png"
+              alt="Bose logo"
+              class="header__logo-img"
+            >
+          </a>
+
+          <div class="header__contacts-menu">
+            <a href="tel:+1 234 555-55-55" class="header__icon icon icon--phone">
+              <div class="icon__tooltip">+1 234 555-55-55</div>
+            </a>
+
+            <a href="#menu" class="header__icon icon icon--menu-opener"></a>
+          </div>
+        </div>
+
+        <h1 class="header__title">
+          The world shades.
+          <br>
+          Your music shines.
+        </h1>
+      </div>
+    </header>
+
+    <nav class="body__nav nav" id="menu">
+      <div class="nav__container">
+        <div class="nav__top">
+          <a href="#" class="nav__logo-link logo">
+            <img
+              src="./images/Logo.png"
+              alt="Bose logo"
+              class="nav__logo-img"
+            >
+          </a>
+          <div class="nav__contacts-menu">
+            <a href="#" class="nav__icon icon icon--menu-close"></a>
+          </div>
+        </div>
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              home
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#recommended" class="nav__link">
+              recommended
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#now-on-view" class="nav__link">
+              now on view
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#buy-direct" class="nav__link">
+              buy direct
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#contacts" class="nav__link">
+              contacts
+            </a>
+          </li>
+        </ul>
+
+        <a href="tel:+1 234 5555-55-55" class="nav__phone-to-call">
+          +1 234 5555-55-55
+        </a>
+
+        <a href="tel:+1 234 5555-55-55" class="nav__call-to-order">
+          call to order
+        </a>
+      </div>
+    </nav>
+
+    <main class="corp">
+      <section class="corp__section recommended" id="recommended" >
+        <h2 class="recommended__title">
+          Recommended
+        </h2>
+
+        <div class="recommended__items-box">
+          <div class="recommended__item">
+            <a href="#item1" class="recommended__buy-photo">
+              <img
+                src="./images/speaker1.png"
+                alt="Bose portable Smart speaker"
+                class="recommended__product-photo"
+              >
+            </a>
+            <a href="#item1" class="recommended__product-name">
+              Bose portable Smart speaker
+            </a>
+            <a href="#smart-home" class="recommended__product-category">
+              Smart home
+            </a>
+            <p class="recommended__buy-price">
+              $ 399.00
+            </p>
```

Comment:
As option you can add "$" and ".00" via before and after pseudo elements, cause when you get the price from server you get only number and then you style it according to mockup